app/js/features/prices/saga.js

```diff
@@ -16,4 +18,23 @@ export function *onFetchPrices() {
   yield takeEvery(FETCH_PRICES, doFetchPrices);
 }
 
-export default [fork(onFetchPrices)];
+
+function *fetchPricesPeriodically() {
```

Comment:
No need to put "periodically" in the function name, since the function itself doesn't do an interval